### PR TITLE
Update AbstractSimplicialComplexes.m2 and SpectralSequences.m2

### PR DESCRIPTION
**Pull request by n-m-g**

### CODE REVIEW

**Comment by d-torrance** [comment]
This PR is using the same branch as #3547 -- closing

File: M2/Macaulay2/packages/SpectralSequences.m2 [hunk 1]
```diff
@@ -1546,325 +1476,111 @@ doc ///
 
 doc ///
     Key
-      "Identifying anti-podal points of the two sphere"
+      "Spectral sequences and non-Koszul syzygies"
```
Comment (mahrud) [inline M2/Macaulay2/packages/SpectralSequences.m2:1479]: Is there just some reordering of the documentation nodes happening here? The differences looks very large so it's hard to tell.

**Comment by n-m-g** [comment]
No, this is a larger update making the package forward compatible with the Complexes update.  So many parts of the code have changed and become reordered in various parts.

Best,
Nathan

**Comment by mahrud** [comment]
I see. I would suggest splitting the package into several smaller files, similar to `NormalToricVarieties` or `Complexes` or `Varieties`, or even much simpler:
```
SpectralSequences.m2
SpectralSequences/docs.m2
SpectralSequences/tests.m2
```

**Comment by n-m-g** [comment]
Many thanks for these suggestions.  Eventually, we may try to do that with future updates.

Best,
Nathan

**Pull request closed without merging**